src/Section.js

```diff
@@ -1,10 +1,33 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import cx from 'classnames';
 
-const Section = props => <div className="section">{props.children}</div>;
+class Section extends Component {
+  render() {
+    const { className, node, id, ...other } = this.props;
+
+    let C = node;
+    let classes = {
+      section: true
+    };
+
+    return (
+      <C {...other} id={id} className={cx(classes, className)}>
```

Comment:
Since we are passing `...other` there's no real need to specify `id` is there?

Reply:
For some reason {...other} doesn't work, at least for me.